en/developer_guide/routing.rst

```diff
@@ -0,0 +1,95 @@
+Routing
+=======
+
+Base URL
+--------
+
+In Open Orchestra, the base url of a website is composed by the domain and eventually a language prefix.
+To manage all possibilities, Open Orchestra associate each website in the database with website aliases which contain:
+
+* **domain**: domain name
+* **language prefix**: use by Open Orchestra to define the language
+* **scheme** : to choose with which protocol page can be access
+
+Here is an example of some base URL generated for a website associate to two domains: ``dev.example.com`` and ``test.example.com``.
+Each of them, with english as default language and french as secondary one.
+
++----------+--------+------------------+----------+-----------------------------------------------+
+| Alias Id | Scheme | Domain           | Language | Language prefix | Base URL                    |
++==========+========+==================+==========+===============================================+
+| 0        | http   | dev.example.com  | fr       | fr              | http://dev.example.com/fr   |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| 1        | http   | dev.example.com  | en       | none            | http://dev.example.com      |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| 2        | http   | test.example.com | fr       | fr              | http://test.example.com/fr  |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| 3        | https  | test.example.com | en       | none            | https://test.example.com    |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+
+
+URL Pattern
+-----------
+
+The url pattern attribute is generated automatically with the name of the page.
```

Comment:
Moreover, the url pattern can be chosen by the contributor, it is only generated if the contributor doesn't give one :)